src/status_im/contexts/keycard/error/view.cljs

```diff
@@ -33,7 +33,7 @@
        :description      :text
        :description-text description}]
      [rn/view {:style {:margin-horizontal 20}}
-      [quo/keycard {:holder-name ""}]
+      [quo/keycard {:blur? true}]
```

Comment:
why :blur? true is needed?

Reply:
Hi @flexsurfer, Thank you very much for reviewing the PR.

We need `:blur? true` because this variant of the keycard component is being used in these screens. Figma [component](https://www.figma.com/design/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?node-id=19119-198284&m=dev) and [screen](https://www.figma.com/design/YGm3igIOAcwMqUVJWCJ6f1/Keycard?node-id=3369-59551&m=dev) design files for reference.

Reply:
thanks